src/erlcloud_as.erl

```diff
@@ -78,6 +82,20 @@
 -define(COMPLETE_LIFECYCLE_ACTION_ACTIVITY, 
         "/CompleteLifecycleActionResponse/ResponseMetadata/RequestId").
 
+-define(COMPLETE_LIFECYCLE_ACTION_RESPONSE,
+    "/CompleteLifecycleActionResponse/ResponseMetadata/RequestId").
+
+-define(RECORD_LIFECYCLE_ACTION_HEARTBEAT_RESPONSE,
+    "/RecordLifecycleActionHeartbeatResponse/ResponseMetadata/RequestId").
+
+
+%% --------------------------------------------------------------------
+%% @doc Sets as hosts in #aws_config{}
+%% @end
+%% --------------------------------------------------------------------
+set_as_host(Host) ->
+    Config = erlcloud_aws:default_config(),
+    put(aws_config, Config#aws_config{as_host = Host}).
```

Comment:
better to pass it once during config and always pass the aws_config{} for all calls.
proc dict method only work for a single basic app inside an application running on ec2